### PR TITLE
bump the version to 1.3.2 to allow fixing an incorrect twine upload of 1.3.1

### LINKED ISSUE #819: Release 1.3.1 is incorrect on pypi
**Describe the bug**

I inadvertently pushed release 1.3.1 to pypi with a migration that was laying around from a dev branch.

**Version**

1.3.1

**Additional context**

I have maintainer but not owner access in pypi so can't delete the incorrect package.

### PR DESCRIPTION
**Pull request by n2ygk**
Fixes #819 

## Description of the Change

No code changes here but a [new version is required by pypi](https://pypi.org/help/#file-name-reuse) due to inadvertently packaging an extra migration that was laying around from a dev branch.

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [ ] documentation updated
- [x] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
   -->
 
+## [1.3.2] 2020-03-24
+
+### Fixed
+* Fixes: 1.3.1 inadvertently uploaded to pypi with an extra migration (0003...) from a dev branch.
```
Comment (auvipy) [inline CHANGELOG.md]: did we reverse that migration?

**Comment by n2ygk** [comment]
Travis succeeded but status got missed: https://travis-ci.org/github/jazzband/django-oauth-toolkit/builds/666529167

**Comment by n2ygk** [comment]
rebased which will hopefully also cause a successful travis status this time.

**Comment by n2ygk** [comment]
@auvipy unstuck travis: please approve

**Pull request merged**